packages/wallets/src/features/cashier/modules/TransactionStatus/TransactionStatus.tsx

```diff
@@ -50,7 +50,7 @@ const TransactionStatus: React.FC<TTransactionStatus> = ({ transactionType }) =>
                 <Text align='start' size='sm' weight='bold'>
                     <Localize i18n_default_text='Transaction status' />
                 </Text>
-                {isError && <LegacyWarningIcon iconSize='xs' />}
+                {isError && <LegacyWarningIcon fill='#FFAD3A' iconSize='xs' />}
```

Comment:
[Q] shouldn't this be var(--text-xxxx)?

Reply:
no they specified hex in figma. Also in the wallets package itself they use hex also.

Reply:
We do use var in most places in wallets as well, you can try this instead:
```suggestion
                {isError && <LegacyWarningIcon fill='var(--status-warning)' iconSize='xs' />}
```